automated install/basic-install.sh

```diff
@@ -1357,27 +1424,61 @@ check_service_active() {
   fi
 }
 
-# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53. 
+# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53.
 disable_resolved_stublistener() {
   echo -en "  ${INFO} Testing if systemd-resolved is enabled"
```

Comment:
This line should have the `-n` flag removed, or the next echos will write from the end of this line instead of on a new line.